secret-providers/gcp-secret-provider/src/main/java/io/camunda/connector/runtime/cloud/GcpSecretManagerSecretProvider.java

```diff
@@ -40,34 +40,29 @@
 
 public class GcpSecretManagerSecretProvider implements SecretProvider {
 
+  /** Secrets used as fallback if SecretProvider is loaded via SPI */
+  public static final String SECRETS_PROJECT_ENV_NAME = "SECRETS_PROJECT_ID";
+
+  public static final String SECRETS_PREFIX_ENV_NAME = "SECRETS_PREFIX";
```

Comment:
is there any changes in this class? If not let's revert the changes (re ordering I guess)

Reply:
Yes thanks to spotless 🤔